Home.md

```diff
@@ -1,40 +1,51 @@
 
-d:swarm is a web application that runs in all modern web browsers. The current release of our web application is available at [http://demo.dswarm.org](http://demo.dswarm.org). If you want to participate in the tests, [drop us a note](mailto:team@dswarm.org), and we will gladly add you to the group of testers. We are looking forward to your feedback, your ideas, your opinion and your contribution at our [mailing list](https://groups.google.com/forum/#!forum/dswarm) or [issue tracker](https://jira.slub-dresden.de).
+d:swarm is realized as a web application that runs in all modern web browsers. The current release of our web application is available at [http://demo.dswarm.org](http://demo.dswarm.org). If you want to participate in the tests, [drop us a note](mailto:team@dswarm.org), and we will gladly add you to the group of testers. We are looking forward to your feedback, your ideas, your opinion and your contribution at our [mailing list](https://groups.google.com/forum/#!forum/dswarm) or [issue tracker](https://jira.slub-dresden.de).
 
-## What can you do with d:swarm?
+## What is the d:swarm idea?
 
-As a base, [this](http://prezi.com/0zh_ypsezu6i/) and [this](http://www.slideshare.net/JensMittelbach/dswarm-a-library-data-management-platform-based-on-a-linked-open-data-approach) presentations summarize the motivation and goals on an abstract level. Furthermore, our [user guide](https://github.com/dswarm/dswarm-documentation/wiki/Overview) provides a brief manual of how to utilise the [d:swarm Backoffice web application](http://demo.dswarm.org).
+Start watching these two ([#1](http://prezi.com/0zh_ypsezu6i/), [#2](http://www.slideshare.net/JensMittelbach/dswarm-a-library-data-management-platform-based-on-a-linked-open-data-approach)) presentations, which summarize the motivation and goals on an abstract level.
 
-### d:swarm is an ETL tool
+## What can I do with d:swarm today?
 
-With the d:swarm ETL tool you can basically complete the following workflow: 
- * how to setup and configure [[data resources|Glossary#data-resource]]
- * create [[projects|Glossary#project]], to define [[mappings|Glossary#mapping]], [[transformations|Glossary#transformation]] and [[filters|Glossary#filter]]
- * explore the [[graph data model|Glossary#graph-data-model]]
- * and export data in [RDF](http://en.wikipedia.org/wiki/Resource_Description_Framework) or XML (e.g. for feeding [Solr](http://lucene.apache.org/solr/) indices).
+With the current d:swarm implementation you can ...
+ * import, set up and configure [[data resources|Glossary#data-resource]]
+ * create [[projects|Glossary#project]], define [[mappings|Glossary#mapping]], [[transformations|Glossary#transformation]] and [[filters|Glossary#filter]]
+ * transform data

```

Comment:
Yes, but we decided to first stress those aspects that represent the currently usable features for a productive use. We may mention it in a less prominent place, though.